pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -302,7 +302,12 @@ func (l *Logging) SetDefaults() error {
 		}
 		if l.Spec.FluentdSpec.FluentOutLogrotate == nil {
 			l.Spec.FluentdSpec.FluentOutLogrotate = &FluentOutLogrotate{
-				Enabled: true,
+				Enabled: false,
+			}
+		}
+		if !l.Spec.FluentdSpec.FluentOutLogrotate.Enabled {
+			if _, ok := l.Spec.FluentdSpec.Annotations["fluentbit.io/exclude"]; !ok {
+				l.Spec.FluentdSpec.Annotations["fluentbit.io/exclude"] = "true"
 			}
 		}
```

Comment:
I was thinking about it, and let's apply this regardless of the logrotate flag. It can still be disabled by setting it to false explicitly just like with syslog-ng

```suggestion
		if _, ok := l.Spec.FluentdSpec.Annotations["fluentbit.io/exclude"]; !ok {
			l.Spec.FluentdSpec.Annotations["fluentbit.io/exclude"] = "true"
		}
```

Reply:
Applied